src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,203 @@
+import java.util.Scanner;
+import duke.task.*;
+import duke.exception.*;
+
+public class Duke {
+
+    private static Scanner sc = new Scanner(System.in);
+    private static Task[] list = new Task[100];
```

Comment:
Consider renaming "list" to make it clearer on what the variable is storing. Perhaps "tasks" would be more suitable?